R/delete_comment_reactions.R

```diff
@@ -33,29 +29,19 @@ delete_comment_reactions <- function(
     comment_id,
     emoji
 ) {
-  assert_string(file_key)
+  file_key <- set_file_key(file_key)
```

Comment:
I think it is reasonable to pull this out for this pull request but I'd really like to see this functionality added to the package overall. I don't see any particular advantage in forcing users to parse file keys and ID values in advance even if a separate utility function is provided.

Reply:
Yes, the intention is to introduce an R6 class as an abstraction layer on top of this 'low-level' API functions.

If we follow this direction it will be nice to create an R6 object responsible for 'scrapping' FIGMA URLs. Then we can use inheritance and create an interface to the R functions that wrap the endpoints. 

This will also overcome any kitchen sink antipattern, while keeping the inputs of the low level API functions consistent with FIGMA API documentation.

Let's move the discussion to a specific issue!